### ISSUE TITLE
Dropdown: add aria attributes

### ISSUE THREAD

**Issue by andrewlarkin** [Comcast/Xooie#16]
Add aria-labelledby attribute to the expander so that when it receives focus it will be described by a descriptive block.  This descriptive block should be tagged using a data attribute.

As will all aria considerations for this module, the attributes should be added with javascript based on the data-role tags leveraged by the modules.
